crates/sui-faucet/src/main.rs

```diff
@@ -41,7 +41,7 @@ struct FaucetConfig {
     #[clap(long, default_value = "127.0.0.1")]
     host_ip: Ipv4Addr,
 
-    #[clap(long, default_value_t = 50000)]
+    #[clap(long, default_value_t = 200_000_000)]
```

Comment:
adjusting this because both locally and on TestNet the default amount should be 5 * 0.2 = 1SUI. On DevNet we are overriding this value